app/workers/postman_worker.rb

```diff
@@ -0,0 +1,8 @@
+class PostmanWorker
+  include Sidekiq::Worker
+
+  def perform(h, count)
```

Comment:
Unused method argument - `count`. If it's necessary, use `_` or `_count` as an argument name to indicate that it won't be used.